### PR TITLE
Adds http_code to heartbeatJSON

### LINKED ISSUE #201: HTTP Status Codes are returned with the text equivalent
**Describe the bug**

The recent introduction of monitoring HTTP status codes allows us to see what the returned code was as part of the `heartbeat` map:

```json
Monitor #1 'Make Monmouth (beta)': Successful Response: 505 ms | Interval: 20 seconds | Type: http
Heartbeat {
  beanMeta: BeanMeta {
    noCache: false,
    fetchAs: '',
    alias: '',
    via: '',
    withCondition: '',
    withConditionData: []
  },
  _monitorId: 1,
  _time: '2021-08-10 06:54:18',
  _status: 1,
  _duration: 20,
  _msg: '200 - OK',
  _ping: 505,
  _important: false
}
```

This is great, however because the `_msg` returns not only the status code `200` but also the ` - OK` response, it makes it difficult to translate this value into something useful for other monitoring systems such as Prometheus.

**Expected behavior**
An additional field could be added to the `heartbeat` so that it returns the following:

```json
Monitor #1 'Make Monmouth (beta)': Successful Response: 505 ms | Interval: 20 seconds | Type: http
Heartbeat {
  beanMeta: BeanMeta {
    noCache: false,
    fetchAs: '',
    alias: '',
    via: '',
    withCondition: '',
    withConditionData: []
  },
  _monitorId: 1,
  _time: '2021-08-10 06:54:18',
  _status: 1,
  _duration: 20,
  _http_status: '200',
 _msg: 'OK',
  _ping: 505,
  _important: false
}
```

This would then allow us to track response time, current state, certificate validation details, **and** the HTTP return codes in the Prometheus end-point.

### PR DESCRIPTION
**Pull request by NiNiyas**
Hopefully closes #201
I don't know anything about anything that I just did, so please test it thoroughly. @louislam @Saibamen @chakflying

### CODE REVIEW

**Review by chakflying — APPROVED** [review]

File: db/patch7.sql [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+create table heartbeat_dg_tmp
+(
+    id INTEGER not null primary key autoincrement,
+    important INTEGER default 0 not null,
+    monitor_id INTEGER default 0 not null,
+    status INTEGER default 0 not null,
+    msg TEXT,
+    time DATETIME not null,
+    ping INTEGER default 0,
```
Comment (chakflying) [inline db/patch7.sql:14]: ping was not default 0 before, did you change it on purpose?

Comment (NiNiyas) [reply db/patch7.sql:14]: No, I didn't. What should it be?

**Comment by louislam** [comment]
The sql file is missing a lot of fields, you have to built the sql on top of patch6.sql.

**Comment by louislam** [comment]
I just re-check the documentation, SQLite allows to add a column by:

```sql
ALTER TABLE employee 
	ADD new_col CHAR(25) DEFAULT '10' NOT NULL
```

It is just not allowed to modify/delete a column. So you can I think you can simplify the sql by this.

https://razorsql.com/features/sqlite_add_column.html

**Comment by NiNiyas** [comment]
Thanks @louislam and @chakflying. I am going to close this rn and give it a go another time. Sorry for wasting your time.

**Pull request closed without merging**